Library/ViewModels/DiscoveryNavigationHeaderViewModel.swift

```diff
@@ -122,7 +122,10 @@ DiscoveryNavigationHeaderViewModelInputs, DiscoveryNavigationHeaderViewModelOutp
     }
 
     self.debugContainerViewIsHidden = self.viewDidLoadProperty.signal
-      .map { !AppEnvironment.current.mainBundle.isAlpha && !AppEnvironment.current.mainBundle.isBeta }
+      .map {
```

Comment:
I find this a little strange to read, we're saying 'hide the debug container _if_ we're not this and not this and not this. Would it be easier to read if it was something like:

"if we're any of these things, don't hide the container"
```
.map { AppEnvironment.current.mainBundle.isDebug || AppEnvironment.current.mainBundle.isAlpha || AppEnvironment.current.mainBundle.isBeta }
.map { !$0 }
```

Reply:
Good point, or maybe even just `.map { AppEnvironment.current.mainBundl.isRelease }` aka "hide the debug menu if the build is release"

Reply:
Ah indeed!